tests/test_Producer.py

```diff
@@ -191,3 +192,17 @@ def test_set_invalid_partitioner_murmur():
     with pytest.raises(KafkaException) as e:
         Producer({'partitioner': 'murmur'})
     assert 'Invalid value for configuration property "partitioner": murmur' in str(e)
+
+
+@pytest.skipif(len([True for x in (".so", ".dylib", ".dll") if os.path.exists("monitoring-interceptor" + x)]) > 0,
```

Comment:
Instead of duplicating this test, move it to a common file (e.g., test_misc) and parameterize the client types: producer, consumer, admin